packages/netlify-cms-core/package.json

```diff
@@ -71,6 +71,7 @@
     "remark-gfm": "1.0.0",
     "sanitize-filename": "^1.6.1",
     "semaphore": "^1.0.5",
+    "semver": "^7.5.4",
```

Comment:
please document why you want semver

Reply:
to conveniently bump the version. will document